bin/ch-test

```diff
@@ -481,6 +482,10 @@ while [[ $# -gt 0 ]]; do
         scope_check "$1"
         scope=$1; shift
         ;;
+    --fmt)
+	require_unset archive_fmt 
+        archive_fmt=$1; shift
```

Comment:
Also support the new equals argument separator (e.g., `--fmt=foo`). If you merge from master, you'll get it for the others and can use that for an example.